animareflection/ui
# Kbd Component

## Description

##### Task link: https://trello.com/c/n1w8Wutg/113-kbd-component

Added `Kbd` (keyboard) component.

## Test Steps

1) Verify structure of component and stories is sound.
2) Verify styles look good.
3) Verify all tests pass.
